core/src/toga/widgets/imageview.py

```diff
@@ -1,13 +1,20 @@
 from __future__ import annotations
 
-from pathlib import Path
+from typing import TYPE_CHECKING
 
 from travertino.size import at_least
 
-from toga.images import Image
+import toga
```

Comment:
I figured that where possible, it would be clearer to `toga.Image` rather than just `Image`, to more clearly distinguish it from `PIL_Image.Image`. The exception I left is in core `window.py`, where that causes an import error.